keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ApplicationSummary.kt

```diff
@@ -0,0 +1,22 @@
+package com.netflix.spinnaker.keel.core.api
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonPropertyOrder
+import com.netflix.spinnaker.keel.api.DeliveryConfig
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder(value = ["name", "application", "serviceAccount", "apiVersion", "paused"])
+data class ApplicationSummary(
+  @JsonIgnore val config: DeliveryConfig,
```

Comment:
I think it would make sense to specify all the properties (`name`, `application`, `serviceAccount`, `apiVersion`) in the data class constructor, then have a secondary constructor that extracts them from a delivery config.